### ISSUE TITLE
Complementary audio and text

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#631]
I thought we were going to drop the statement about embedded audio clips from the ways of reading section, but at the least I think the compact description needs rewriting. The descriptive version says:

> Portions of prerecorded audio are available

That's significantly different from saying there is complementary text for the audio. Looking at the epub techniques, the test only checks if there are audio sources based on there being a primary audio access mode, not whether there is any text that complements it.

**Comment by clapierre** [comment]
Right in EPUB Techniques in the ways to read we have

ELSE IF audio_content:
THEN display "Complementary audio and text"[ID: ways-of-reading-prerecorded-audio-complementary].

Which I think is fine to be in Ways of Reading as this is correct, just means you will have some auditory in the book and there is text which is not synchronized the audio could be anything not really associated with the text. 
However, what should the string be?  "Complementary audio and text" vs. "Portions of prerecorded audio are available".

**Comment by mattgarrish** [comment]
Complementary audio and text sounds like you're saying there are text equivalents. One complements the other. That's the disconnect I have between it and the descriptive which isn't saying anything about whether there is text or not.

What if the strings were something like:

> Prerecorded audio clips
> Prerecorded audio clips are embedded in the content

Is that what we want to say here?

**Comment by clapierre** [comment]
I can see that confusion. I like those two options would one be the short and the other the descriptive version or this was our two options for the simple short description?

**Comment by mattgarrish** [comment]
I was thinking one compact and the other descriptive. I can't think of really different ways of phrasing this.

**Comment by clapierre** [comment]
Work for me, once the Alphabetical PR #624 has been merged I can make the EPUB Technique change to change the string.

**Comment by GeorgeKerscher** [comment]
I agree.  Why don't we make the descriptive and compact the same?

**Comment by clapierre** [comment]
Fixed in EPUB Techniques.

**Comment by GeorgeKerscher** [comment]
Fixed in guidelines-jan-30

**Comment by HadrienGardeur** [comment]
> we're not currently testing whether synchronized text and audio is available for part of the text (`accessModeSufficient` doesn't have `auditory` on its own) or the entire text (`accessModeSufficient` has `auditory` on its own)

Should I open a separate issue about this?
